### DISCUSSION TITLE
What if ImplementingTypeDefinition.class is passed?

### DISCUSSION THREAD

**Discussion by nikolayandr** [graphql-java/graphql-java#2623] [Ideas]
https://github.com/graphql-java/graphql-java/blob/4eed7c96fb8f394a633a361f6645eb8d87df5bb0/src/main/java/graphql/schema/idl/TypeDefinitionRegistry.java#L471

**graphql.schema.idl.TypeDefinitionRegistry#getType(java.lang.String, java.lang.Class<T>)** can't be used for cases when it is just necessary to get ImplementingTypeDefinition no matter whether it is interface or object

**Comment by bbakerman** [comment]
PRs are welcome
